theano/tensor/nnet/abstract_conv.py

```diff
@@ -1458,6 +1461,11 @@ def __init__(self, convdim,
         if len(filter_dilation) != convdim:
             raise ValueError("filter_dilation must have {} elements".format(convdim))
         self.filter_dilation = tuple(filter_dilation)
+        if num_groups < 1:
+            raise ValueError("num_groups must have value greater than zero")
+        elif num_groups > 1 and convdim == 3:
+            raise ValueError("grouped convolution not supported for 3D convolutions")
+        self.num_groups = num_groups
 
     def do_constant_folding(self, node):
         # Disable constant folding since there is no implementation.
```

Comment:
note that the flops method bellow will need to be adjusted for groups. But do this later, when the rest work. For now, just make it raise an error when groups isn't 1.